application/common/models/PasswordChangeLogBase.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace common\models;
+
+use Yii;
+
+/**
+ * This is the model class for table "password_change_log".
+ *
+ * @property integer $id
+ * @property integer $user_id
+ * @property string $scenario
+ * @property string $reset_type
+ * @property string $method_type
+ * @property string $masked_value
+ * @property string $created
+ * @property string $ip_address
+ *
+ * @property User $user
+ */
+class PasswordChangeLogBase extends \yii\db\ActiveRecord
+{
+    /**
+     * @inheritdoc
+     */
+    public static function tableName()
+    {
+        return 'password_change_log';
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function rules()
+    {
+        return [
+            [['user_id', 'scenario', 'reset_type', 'method_type', 'masked_value', 'created', 'ip_address'], 'required'],
+            [['user_id'], 'integer'],
+            [['scenario', 'reset_type', 'method_type'], 'string'],
+            [['created'], 'safe'],
```

Comment:
I'm not sure if there's a simple way to change this, but all of these `created` columns should probably only be `safe` on insert (not update), since we wouldn't want the created value to be changed.